client/handler.go

```diff
@@ -221,6 +297,64 @@ func (h *Handler) updateClient(ctx context.Context, c *Client) error {
 	return nil
 }
 
+// swagger:route PUT /dyn-clients/{id} public updateDynOAuth2Client
```

Comment:
Make the client ID part of the query string as explained shown here: https://openid.net/specs/openid-connect-registration-1_0.html#ReadRequest

```suggestion
// swagger:route PUT /connect/register public updateOAuth2ClientPublic
```